worker/src/RTC/RtpRetransmissionBuffer.cpp

```diff
@@ -392,14 +380,24 @@ namespace RTC
 	{
 		MS_TRACE();
 
-		return this->Get(this->startSeq);
+		if (this->buffer.empty())
+		{
+			return nullptr;
+		}
+
+		return this->buffer.front();
```

Comment:
Oldest seq in the buffer is just the first element by design, period. Let's do it easy.